### PR TITLE
Add io.github.davidoc26.wallpaper_selector

### PR DESCRIPTION
**Pull request by Davidoc26**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
--talk-name=org.freedesktop.Flatpak - Used to set desktop wallpaper via cli. For example to use gsettings
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Davidoc26** [comment]
bot, build io.github.davidoc26.wallpaper_selector

File: io.github.davidoc26.wallpaper_selector.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "wallpaper-selector",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Flatpak"
```
Comment (hfiguiere) [inline io.github.davidoc26.wallpaper_selector.json:16]: why do you need this DBus access?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.github.davidoc26.wallpaper_selector.json (hunk 1 above)
Comment (Davidoc26) [reply io.github.davidoc26.wallpaper_selector.json:16]: As I wrote, this is necessary in order to set the desktop wallpaper via cli, for example using gsettings in gnome

Comment (barthalion) [reply io.github.davidoc26.wallpaper_selector.json:16]: There's a bunch of wallpaper apps like https://beta.flathub.org/apps/details/im.bernard.Nostalgia which seem to use libportal for setting wallpaper.

**Comment by Davidoc26** [comment]
bot, build io.github.davidoc26.wallpaper_selector

**Pull request closed without merging**